components/Footer.jsx

```diff
@@ -0,0 +1,3 @@
+function Footer() {
+  return (<footer>Footer Content</footer>);
```

Comment:
Codacy has a fix for the issue: [Replace `(<footer>Footer·Content</footer>)` with `<footer>Footer·Content</footer>`](https://app.codacy.com/gh/thamizhMsys/testRepo6/pullRequest?prid=9650719)
```suggestion
  return <footer>Footer Content</footer>;
```